README.md

```diff
@@ -1 +1,5 @@
-# html-starter
\ No newline at end of file
+# html-starter 
+
+Just edit the *index.html* and *style.css* files to begin writing you're website/.
+
+Open it directly in browser from a file manager or use the built-in feature in Webstorm by click the browser logo in the upper right of index.html.
```

Comment:
Replace this line with:

Open it directly in browser from a file manager by double clicking on index.html or use the built-in feature in Webstorm by click the browser logo in the upper right of index.html.